src/network/message_blockdata.rs

```diff
@@ -31,7 +31,11 @@ pub enum InvType {
     /// Transaction
     Transaction,
     /// Block
-    Block
+    Block,
+    /// Witness Block
+    WitnessBlock,
+    /// Witness Transaction
+    WitnessTransaction
```

Comment:
Style nit: can you add a trailing `,` as per recent Rust style guidelines, and to simplify further diffs?

I'll merge this as is because it's not important, just for future.